initramfs/debian/rules

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@ 
+
+override_dh_auto_test:
+	@set -e; for f in scripts/*; do \
+	    echo "Checking shell syntax of $$f"; \
+	    sh -n $$f; \
```

Comment:
i have a branch half ready that uses shellcheck on commits, i didnt want to add shellcheck (universe) to the build deps because that would make a possible MIR harder ...